src/nginxconfig/templates/domain_sections/restrict.vue

```diff
@@ -0,0 +1,256 @@
+                </div>
+                <div class="field is-horizontal">
+                    <div class="field-label">
+                        <label class="label">TRACE</label>
+                    </div>
+                    <div class="field-body">
+                        <div class="field">
+                            <div :class="`control${traceMethodChanged ? ' is-changed' : ''}`">
+                                <div class="checkbox">
+                                    <PrettyCheck v-model="traceMethod" class="p-default p-curve p-fill p-icon">
+                                        <i slot="extra" class="icon fas fa-check"></i>
+                                        {{ i18n.templates.domainSections.restrict.disableForThisDomain }}
+                                    </PrettyCheck>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    import PrettyCheck from 'pretty-checkbox-vue/check';
+    import i18n from '../../i18n';
+    import delegatedFromDefaults from '../../util/delegated_from_defaults';
+    import computedFromDefaults from '../../util/computed_from_defaults';
+
+    const defaults = {
+        getMethod: {
+            default: false,
+            enabled: true,
+        },
+        postMethod: {
+            default: false,
+            enabled: true,
+        },
+        putMethod: {
+            default: false,
+            enabled: true,
+        },
+        patchMethod: {
+            default: false,
+            enabled: true,
+        },
+        deleteMethod: {
+            default: false,
+            enabled: true,
+        },
+        headMethod: {
+            default: false,
+            enabled: true,
+        },
+        connectMethod: {
+            default: false,
+            enabled: true,
+        },
+        optionsMethod: {
+            default: false,
+            enabled: true,
+        },
+        traceMethod: {
+            default: false,
+            enabled: true,
+        },
+    };
+
+    export default {
+        name: 'DomainRestrict',                                  // Component name
+        display: i18n.common.restrict,                           // Display name for tab
+        key: 'restrict',                                         // Key for data in parent
+        delegated: delegatedFromDefaults(defaults),             // Data the parent will present here
+        components: {
+            PrettyCheck,
+        },
+        props: {
+            data: Object,                                       // Data delegated back to us from parent
+        },
+        data () {
+            return {
+                i18n,
+            };
+        },
+        computed: {
+            ...computedFromDefaults(defaults, 'restrict'),    // Getters & setters for the delegated data
+        },
```

Comment:
```suggestion
        computed: computedFromDefaults(defaults, 'restrict'),   // Getters & setters for the delegated data
```
Can be a single line :)

Reply:
I've actually added another computed property in newest commit.